src/main/java/nextstep/ladder/domain/LadderResults.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.ladder.domain;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LadderResults {
+    private final Map<Name, Name> value;
```

Comment:
👍 
조금 더 직관적인 네이밍을 활용해 보셔도 좋을 것 같아요.
```suggestion
    private final Map<Participant, Reward> value;
```